src/providers/stockpile-data.ts

```diff
@@ -30,7 +30,14 @@ export class StockpileData {
 
   showToast(message: string) {
     if (this.platform.is('cordova')) {
-      Toast.show(message, '5000', 'bottom').subscribe(toast => {});
+      Toast.showWithOptions(
+        {
+          message: message,
```

Comment:
I'm not sure if this is ES2015 or a future version, but you may be able to shorten this line to just:
```JavaScript
message,
```
I use this often on the backend when assigning object properties with the same name as the variable that holds their values. Worth a shot!